### ISSUE TITLE
routeType='pt' not working for onemap.route()

### ISSUE THREAD

**Issue by imoea** [windspeed-io/python-onemapsg#6]
The JSON object returned by the API for `routeType='pt'` doesn't follow the convention assumed in your package and so `route.to_dict()` returns an empty dictionary. The result is actually stored under the key `'plan'`.

Suggested fix in `response.py` (see the last line):

```
class RouteResult(BaseResource):

    status_message = None
    alternative_names = None
    route_name = None
    route_geometry = None
    route_instructions = None
    alternative_summaries = None
    via_points = None
    route_summary = None
    found_alternative = None
    status = None
    via_indices = None
    hint_data = None
    alternative_geometries = None
    alternative_instructions = None
    alternative_indices = None
    plan = None
```

**Comment by thomasjiangcy** [comment]
Thanks for spotting this! Would you like to open a pull request for this?

**Comment by thomasjiangcy** [comment]
Resolved by #15 #
